tools/probes/probes_main.c

```diff
@@ -267,45 +267,34 @@ void parse_data(char *file_in)
 	memset(&data, 0, DATA_READ_LIMIT);
 	memset(&files, 0, sizeof(struct wave_files) * FILES_LIMIT);
 
+	start = 0;
 	/* data read loop to process DATA_READ_LIMIT bytes at each iteration */
```

Comment:
Is the comment still valid when start is added?

Reply:
Pretty much, if not counting the minor exception of moving 1-3 bytes remainder to the beginning of buffer  if looking for the next sync word... Anyway updated the comment.